README.md

```diff
@@ -381,12 +388,12 @@ audible artifacts.
 
 The `-I` parameter disables the described catch-up behaviour and instead
 resyncs to the stream after an interruption i.e. continues to play the
-later received ETI frames in realtime. However this means that the
+later received ETI frames in realtime. However, this means that the
 playback is delayed by the amount of all previous interruptions i.e. the
 news will start some seconds/minutes later compared to live reception
 because of that.
 
-The GTK GUI version also allows to stop decoding the current
+The GTK GUI version (dablin_gtk) also allows stopping decoding the current
```

Comment:
Hmm, what is wrong with "to stop" here?

Reply:
No, the gerund has to be used here, see https://www.grammaring.com/to-infinitive-or-gerund-advise-recommend-allow-permit-forbid-require as there is no object.

... allows **us** to stop ... would be correct on the other hand.

Reply:
Ah, this indeed wrong in the current version. However, "stopping decoding" sounds a bit awkward...."allows **a user** to stop" would be a good alternative.